Source/Core/Core/DSP/Jit/x64/DSPEmitter.h

```diff
@@ -206,10 +186,28 @@ class DSPEmitter final : public JIT::DSPEmitter, public Gen::X64CodeBlock
   void msub(UDSPInstruction opc);
 
 private:
+  using DSPCompiledCode = u32 (*)();
+  using Block = const u8*;
+
   // The emitter emits calls to this function. It's present here
   // within the class itself to allow access to member variables.
   static void CompileCurrent(DSPEmitter& emitter);
 
+  u16 RunCycles(u16 cycles) override;
+  void DoState(PointerWrap& p) override;
+  void ClearIRAM() override;
```

Comment:
It's allowed, but does it actually make sense to change the visibility of those 3 to `private` (vs. `public` from `JIT::DSPEmitter`) in here?